src/app/permissions/authorized-users-select/authorized-users-select.component.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { async, ComponentFixture, discardPeriodicTasks, fakeAsync, TestBed, tick } from '@angular/core/testing';
+
+import { AuthorizedUsersSelectComponent } from './authorized-users-select.component';
+import { UsersService } from '../../users/users.service';
+import { of } from 'rxjs';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { LayoutModule } from '@angular/cdk/layout';
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { ReactiveFormsModule } from '@angular/forms';
+import { AppMaterialModule } from '../../app-material.module';
+import { User } from '../../users/user';
+import { MatAutocompleteSelectedEvent } from '@angular/material';
+
+const userSuggestions: User[] = [
+  {
+    id: '2736a204-f3ab-4b65-8568-a1c8ce1db8ab',
+    userName: 'testing',
+    firstName: 'Tina',
+    lastName: 'Testing',
+    email: 'tina.testing@skoop.io',
+    coach: false,
+  }
+];
+
+describe('AuthorizedUsersSelectComponent', () => {
+  let component: AuthorizedUsersSelectComponent;
+  let fixture: ComponentFixture<AuthorizedUsersSelectComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        BrowserAnimationsModule,
+        LayoutModule,
+        FlexLayoutModule,
+        ReactiveFormsModule,
+        AppMaterialModule
+      ],
+      declarations: [ AuthorizedUsersSelectComponent ],
+      providers: [
+        {
+          provide: UsersService,
+          useValue: jasmine.createSpyObj('userService', {
+            'getAuthorizedUsers': of<User[]>(
+              [{
```

Comment:
Can we return from `getAuthorizedUsers` the value `userSuggestions` as well as soon as there is no difference between results of `getUserSuggestions` and `getAuthorizedUsers`?